app/Kconfig.behaviors

```diff
@@ -1,17 +1,87 @@
 # Copyright (c) 2023 The ZMK Contributors
 # SPDX-License-Identifier: MIT
 
+config ZMK_BEHAVIOR_BACKLIGHT
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_BACKLIGHT_ENABLED && ZMK_BACKLIGHT
+
+config ZMK_BEHAVIOR_BLUETOOTH
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_BLUETOOTH_ENABLED && ZMK_BLE
+
+config ZMK_BEHAVIOR_CAPS_WORD
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_CAPS_WORD_ENABLED
+
+config ZMK_BEHAVIOR_EXT_POWER
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_EXT_POWER_ENABLED && ZMK_EXT_POWER
+
+config ZMK_BEHAVIOR_HOLD_TAP
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_HOLD_TAP_ENABLED
+
+config ZMK_BEHAVIOR_KEY_PRESS
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_KEY_PRESS_ENABLED
+
+config ZMK_BEHAVIOR_KEY_REPEAT
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_KEY_REPEAT_ENABLED
+
 config ZMK_BEHAVIOR_KEY_TOGGLE
     bool
     default y
     depends on DT_HAS_ZMK_BEHAVIOR_KEY_TOGGLE_ENABLED
 
+config ZMK_BEHAVIOR_MACRO
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_MACRO_ENABLED || DT_HAS_ZMK_BEHAVIOR_MACRO_ONE_PARAM_ENABLED || DT_HAS_ZMK_BEHAVIOR_MACRO_TWO_PARAM_ENABLED
+
+config ZMK_BEHAVIOR_MOD_MORPH
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_MOD_MORPH_ENABLED
+
+config ZMK_BEHAVIOR_MOMENTARY_LAYER
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_MOMENTARY_LAYER_ENABLED
+
 config ZMK_BEHAVIOR_MOUSE_KEY_PRESS
     bool
     default y
     depends on DT_HAS_ZMK_BEHAVIOR_MOUSE_KEY_PRESS_ENABLED
     imply ZMK_MOUSE
 
+config ZMK_BEHAVIOR_NONE
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_NONE_ENABLED
+
+config ZMK_BEHAVIOR_OUTPUTS
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_OUTPUTS_ENABLED
+
+config ZMK_BEHAVIOR_RESET
+    bool
+    default y
+    depends on DT_HAS_ZMK_BEHAVIOR_RESET_ENABLED
```

Comment:
So this one is a bit tricky... This is needs to be enables a bit more specifically for the right side of splits, even if it's not referenced in a keymap. There's a few of those, like the RBG behavior, that are either "local" or "global" so need this special handling.

Reply:
This makes sense. I've removed the Kconfig options for the behaviors we talked about in Discord and reverted them back to using the main Kconfig option where available. This means the reset behaviors are always compiled in for both sides.